src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -107,42 +105,42 @@ public static List<PartyDetails> buildHearingPartiesDetails(SscsCaseData caseDat
                 OtherParty otherParty = ccdOtherParty.getValue();
                 partiesDetails.addAll(buildHearingPartiesPartyDetails(
                         otherParty, otherParty.getRep(), otherParty.getHearingOptions(),
-                        otherParty.getHearingSubtype(), appellant.getId(), null, referenceDataServiceHolder));
+                        otherParty.getHearingSubtype(), null, referenceDataServiceHolder, null));
             }
         }
 
         return partiesDetails;
     }
 
-    public static List<PartyDetails> buildHearingPartiesPartyDetails(Party party, String appellantId, ReferenceDataServiceHolder referenceData) throws InvalidMappingException {
-        return buildHearingPartiesPartyDetails(party, null, null, null, appellantId, null, referenceData);
+    public static List<PartyDetails> buildHearingPartiesPartyDetails(Party party, String appellantId, ReferenceDataServiceHolder referenceData, String adjournLanguage) throws InvalidMappingException {
```

Comment:
line is too long (check the rest of this file for them as theres a few it seems

Reply:
fixed